### PR TITLE
ridho/AuthorisedApplication

### PR DESCRIPTION
**Pull request by ridhomard**
Approve Authorised
Cancel Authorised
Check authorised page
Revoke authorised

### CODE REVIEW

File: src/test/java/testCases/Validate_AuthorisedApplications.java [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+        AuthorisedApplications_Action.cancelPermission(driver);
+        Login_Action.Execute(driver,Constant.Email,Constant.Password);
+    }   
+    //Test method to check grant permission page
+    @Test(priority=7)
+    public void grant() {
+        AuthorisedApplications_Action.grantPermission(driver);
+    }
+
+    //Test Method to close browser session
```
Comment (bijan2007) [inline src/test/java/testCases/Validate_AuthorisedApplications.java]: Please correct the comments

File: src/test/java/testCases/Validate_AuthorisedApplications.java [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+import static org.testng.Assert.assertTrue;
+
+import java.util.concurrent.TimeUnit;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeTest;
+
```
Comment (bijan2007) [inline src/test/java/testCases/Validate_AuthorisedApplications.java]: Please cleanup the spaces between import statements

File: src/test/java/appModules/AuthorisedApplications_Action.java [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    public static  void revoke(WebDriver driver) {
+        AuthorisedApplications_page.revoke_button(driver).click();
+
+    }
+    //Method to check review page is loaded
+    public static  void reviewPage(WebDriver driver) {
+        Assert.assertTrue(AuthorisedApplications_page.review_title(driver).isDisplayed());
+        Assert.assertTrue(AuthorisedApplications_page.scope_list(driver).isDisplayed());
+    }
+    public static  void cancelPermission(WebDriver driver) {
```
Comment (bijan2007) [inline src/test/java/appModules/AuthorisedApplications_Action.java]: Please add comments for the methods

**Comment by ridhomard** [comment]
Thanks @bijan2007 .. fixed it .

**Pull request merged**